src/layout/header/Menu.tsx

```diff
@@ -80,11 +80,11 @@ const Menu: React.FC<MenuProps> = ({ menuButtonLabel, handleLogout }) => {
             handleClose();
             history.push('/user');
           }}>
-          {t('My profile')}
+          {t('profile:my_profile')}
         </StyledMenuItem>
 
         <StyledMenuItem onClick={handleLogout} data-testid="logout-button">
-          {t('Logout')}
+          {t('logout')}
```

Comment:
`profile:`? evt inn i hooken

Reply:
login og logout ligger i translations.json nå. usikker på om vi trenger en egen fil for header-oversettelser. men tenker vi kan ta det senere når det blir flere ting som skal oversettes der